### PR TITLE
Update cis-compliance.md

### PR DESCRIPTION
**Pull request by JustinN-Replicated**
@jala-dx Making this adjustment per Marc's comment:
> Can we document what the adverse effect are?

I am presuming that the kURL node could 'break' but I am not  aware of the details of how/why

### CODE REVIEW

**Review by jala-dx — APPROVED** [review]

**Comment by JustinN-Replicated** [comment]
@jala-dx we are looking to address Marc's comment as I commetned above; my change is likely insufficient.

Also, on line 38 from Marc here https://github.com/replicatedhq/kurl.sh/pull/750#discussion_r834360490 :
> It feels like this field should be under kubernetes not kurl? Or are we ensuring that all add-ons are CIS compliant also?

 iirc it is because we may need this check for other distros yeah? AFAIK we do not check add-ons

**Comment by JustinN-Replicated** [comment]
@jala-dx I noticed you had commented on #750 - we should likely focus on this PR since that one is closed. Did you have a chance to review the above comments? Let me know how I can help

**Comment by jala-dx** [comment]
> @jala-dx we are looking to address Marc's comment as I commetned above; my change is likely insufficient.
> 
> Also, on line 38 from Marc here [#750 (comment)](https://github.com/replicatedhq/kurl.sh/pull/750#discussion_r834360490) :
> 
> > It feels like this field should be under kubernetes not kurl? Or are we ensuring that all add-ons are CIS compliant also?
> 
> iirc it is because we may need this check for other distros yeah? AFAIK we do not check add-ons

@marccampbell recommendation `It feels like this field should be under kubernetes not kurl?` is taken care with https://github.com/replicatedhq/kURL/pull/2783 .

**Review by jala-dx — APPROVED** [review]

File: src/markdown-pages/install-with-kurl/cis-compliance.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ spec:
 ## Known Limitations
 
 * The [EKCO add-on](/docs/add-ons/ekco) v0.19.0 and later is required to use this feature.
-* This feature works with the [Kubernetes (kubeadmn) add-on](https://kurl.sh.docs/add-ons/kubernetes) only.
+* This feature works with the [Kubernetes (kubeadm) add-on](https://kurl.sh/docs/add-ons/kubernetes) only.
 * To meet CIS compliance, admin.conf permissions are changed from the default `root:sudo 440` to `root:root 444`.
 * Kubelet no longer attempts to modify non-standard kernel flags. Using non-standard kernel flags can block the Kubelet from initializing and causes the installation to fail.
-* This feature is not supported for upgrades of existing kURL installations because the settings that are changed can introduce adverse effects.
+* This feature is not supported for upgrades of existing kURL installations because the settings that are changed can introduce adverse effects and render the existing kURL node unusuable.
```
[the remark this answers is not in the document]
Comment (JustinN-Replicated) [reply src/markdown-pages/install-with-kurl/cis-compliance.md:67]: Just discussed with Jalaja, this is no longer the case - see upgrades on testgrid here: https://testgrid.kurl.sh/run/full-run-after-kotsadm

File: src/markdown-pages/install-with-kurl/cis-compliance.md [hunk 2]
```diff
@@ -62,7 +61,16 @@ spec:
...
+* This feature has been tested with kURL upgrades however it is strongly suggested to test this with your development environments first before upgrading production.
+
+## AWS Amazon Linux 2 (AL2) Considerations
+The kernel defaults of this Amazon Machine Image (AMI) are not set properly for CIS compliance. CIS compliance does not allow Kubernetes to change kernel settings itself. It is required that the following kernel defaults must be set prior to installing with kURL:
+``` bash 
+sudo sysctl vm.overcommit_memory=1
+sudo sysctl kernel.panic=10
+sudo sysctl kernel.panic_on_oops=1
+```
+Failure to set these values will result in kernel panics. These settings must also be configured on AL2 instance nodes prior to upgrading them to a CIS compliant kURL specification.
```
Comment (laverya) [inline src/markdown-pages/install-with-kurl/cis-compliance.md:76]:
```suggestion
Failure to set these values will result in kubelet crashing.
These settings must also be configured on AL2 instance nodes prior to upgrading them to a CIS compliant kURL specification. 
```

File: src/markdown-pages/install-with-kurl/cis-compliance.md [hunk 3]
```diff
@@ -62,7 +61,16 @@ spec:
 ## Known Limitations
 
 * The [EKCO add-on](/docs/add-ons/ekco) v0.19.0 and later is required to use this feature.
-* This feature works with the [Kubernetes (kubeadmn) add-on](https://kurl.sh.docs/add-ons/kubernetes) only.
+* This feature works with the [Kubernetes (kubeadm) add-on](https://kurl.sh/docs/add-ons/kubernetes) only.
 * To meet CIS compliance, admin.conf permissions are changed from the default `root:sudo 440` to `root:root 444`.
 * Kubelet no longer attempts to modify non-standard kernel flags. Using non-standard kernel flags can block the Kubelet from initializing and causes the installation to fail.
```
Comment (laverya) [inline src/markdown-pages/install-with-kurl/cis-compliance.md:66]:
```suggestion
* Kubelet no longer attempts to change kernel parameters at runtime. Using kernel parameters other than those expected by kubernetes can block Kubelet from initializing and causes the installation to fail.
```

**Review by jala-dx — DISMISSED** [review]

**Comment by JustinN-Replicated** [comment]
with most recent changes lgtm

**Review by jonquil2002 — APPROVED** [review]
Committed some changes. Otherwise, lgtm.

**Pull request merged**